components/server/src/orgs/default-workspace-image-validator.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+export type DefaultWorkspaceImageValidator = (userId: string, imageRef: string) => Promise<void>;
```

Comment:
I moved defaulta wokrspace image validation to organizaion-service, but then it depends on workspace-service. So decouple it we use an intermediate service with lazy loading.